### PR TITLE
Map some BE renderers

### PR DESCRIPTION
**Pull request by Tom-The-Geek**

### CODE REVIEW

File: mappings/net/minecraft/client/render/block/entity/BannerBlockEntityRenderer.mapping [hunk 1]
```diff
@@ -11,4 +11,14 @@ CLASS net/minecraft/class_823 net/minecraft/client/render/block/entity/BannerBlo
 		ARG 5 baseSprite
 		ARG 6 isBanner
 		ARG 7 patterns
+		ARG 8 glint
+	METHOD method_29999 renderCanvas (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IILnet/minecraft/class_630;Lnet/minecraft/class_4730;ZLjava/util/List;)V
+		ARG 0 matrices
+		ARG 1 vertextConsumers
```
Comment (YanisBft) [inline mappings/net/minecraft/client/render/block/entity/BannerBlockEntityRenderer.mapping]:
```suggestion
		ARG 1 vertexConsumers
```

File: mappings/net/minecraft/client/render/block/entity/BedBlockEntityRenderer.mapping [hunk 2]
```diff
@@ -1,7 +1,12 @@
 CLASS net/minecraft/class_825 net/minecraft/client/render/block/entity/BedBlockEntityRenderer
+	FIELD field_27744 bedHead Lnet/minecraft/class_630;
+	FIELD field_27745 bedFoot Lnet/minecraft/class_630;
 	METHOD method_32136 getHeadTexturedModelData ()Lnet/minecraft/class_5607;
 	METHOD method_32137 getFootTexturedModelData ()Lnet/minecraft/class_5607;
-	METHOD method_3558 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_630;Lnet/minecraft/class_2350;Lnet/minecraft/class_4730;IIZ)V
+	METHOD method_3558 renderPart (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_630;Lnet/minecraft/class_2350;Lnet/minecraft/class_4730;IIZ)V
 		ARG 1 matrix
+		ARG 2 vertextConsumers
```
Comment (YanisBft) [inline mappings/net/minecraft/client/render/block/entity/BedBlockEntityRenderer.mapping]:
```suggestion
		ARG 2 vertexConsumers
```

File: mappings/net/minecraft/client/render/block/entity/EndPortalBlockEntityRenderer.mapping [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+		ARG 2 model
+		ARG 3 vertices
+		ARG 4 x1
+		ARG 5 x2
+		ARG 6 y1
+		ARG 7 y2
+		ARG 8 z1
+		ARG 9 z2
+		ARG 10 z3
+		ARG 11 x4
```
Comment (YanisBft) [inline mappings/net/minecraft/client/render/block/entity/EndPortalBlockEntityRenderer.mapping]:
```suggestion
		ARG 11 z4
```

**Review by YanisBft — COMMENTED** [review]
Looks good!

**Comment by Tom-The-Geek** [comment]
Whoops! I thought I'd checked it all for typos

**Review by YanisBft — APPROVED** [review]
Mind fixing the conflicts as well?

**Comment by Tom-The-Geek** [comment]
I think that fixes the conflicts, I've never really done that before.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**